include/deal.II/base/index_set.h

```diff
@@ -344,6 +344,14 @@ class IndexSet
   IndexSet
   get_view(const size_type begin, const size_type end) const;
 
+  /**
+   * Split the set of DoFs (typically locally owned or relevant) in @p whole_set into blocks
```

Comment:
To be concrete, @masterleinad suggests to change this as follows:
```suggestion
   * Split the set indices represented in @p whole_set into blocks
```
and also getting rid of the term "degrees of freedom" (or dofs) in the text below.